src/main/java/com/exasol/repository/GitBranchContent.java

```diff
@@ -6,6 +6,13 @@
  * This class represents Git repository content based on the latest commit of the user-specified branch.
  */
 public interface GitBranchContent {
```

Comment:
Why is it named `GitBranchContent` and not `GitBranch`?

Reply:
Because it represented only the content so far. With the new method, I've just added I also have doubts now if I should rename it or not. Maybe I'll rename it to GitBranch or divide content and branch logic